internal/logging/logging.go

```diff
@@ -0,0 +1,23 @@
+package logging
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+// SdkEventsToLogArguments converts a given sdk.Events and returns a slice of strings that provide human
+// readable values for the event attributes.
+func SdkEventsToLogArguments(events sdk.Events) []string {
+	logArgs := []string{"events"}
+	for _, e := range events {
+		logArgs = append(logArgs, fmt.Sprintf("type=%s", e.Type))
+		for _, attr := range e.Attributes {
+			if len(attr.Value) == 0 {
+				continue
+			}
+			logArgs = append(logArgs, fmt.Sprintf("%s=%s", attr.Key, attr.Value))
+		}
+	}
+	return logArgs
+}
```

Comment:
Thanks for reviewing guys! I noticed that when events were logged they were being displayed as raw bytes and not strings. I added this function to make them human readable.

Note: This is introducing an `internal` package since this function doesn't really belong to a single module, and is not likely going to be imported from anyone using ibc-go.

cc @charleenfei @damiannolan

Reply:
is it posible to use `sdk.StringifyEvents`?

https://github.com/cosmos/cosmos-sdk/blob/main/types/events.go#L331

Reply:
After looking into this, the output includes lots of indentation

```
emitting interchain account registration events module=x/ibc-interchainaccounts events="\t\t- channel_open_init\n\t\t\t- port_id: icacontroller-cosmos1tnpsf8r0lk9uct8mas333auhytjuq2kz0372at\n\t\t\t- channel_id: channel-0\n\t\t\t- counterparty_port_id: icahost\n\t\t\t- counterparty_channel_id: \n\t\t\t- connection_id: connection-0\n\t\t\t- version: {\"version\":\"ics27-1\",\"controller_connection_id\":\"connection-0\",\"host_connection_id\":\"connection-0\",\"address\":\"\",\"encoding\":\"proto3\",\"tx_type\":\"sdk_multi_msg\"}\n\t\t- message\n\t\t\t- module: ibc_channel"
```

This looks like it is maybe for the cli output